### ISSUE TITLE
Align exercises when math is present on first line in the EOC

### ISSUE THREAD

**Issue by helenemccarron** [openstax/cnx-recipes#1370]
Currently (assuming `math {vertical-align: middle}`:
![2019-09-09_14-37-42](https://user-images.githubusercontent.com/8514591/64561061-71479e80-d30f-11e9-96dc-28988ac1d872.png)

Suggested CSS:
```.os-eoc.os-exercises-container {
  [data-type="problem"] {
    display: table;
    .os-number {color: #000000; font-weight: bold; text-decoration: none; display: table-cell;}
    .os-divider {padding-right: 8px; display: table-cell;}
    .os-problem-container {display: table-cell;}
  }
}
```

Result: 
![2019-09-09_14-31-20](https://user-images.githubusercontent.com/8514591/64561145-a227d380-d30f-11e9-88ff-4d125627bea2.png)

**Comment by openstaxalina** [comment]
This may slide to the U.U sprint.

**Comment by brittany-johnson** [comment]
Update: When building the PDF with the mathified content, the math is aligned. The issue now is that it is bolded and there is almost no space on either side of the SVG. 

I've tried changing the font-weight to many different values but I can't get it to apply in the PDF. I am able to change the color and font size for example, so I'm not sure why I can't change the weight. 
<img width="1233" alt="Screen Shot 2019-11-06 at 10 00 05 AM" src="https://user-images.githubusercontent.com/24909007/68314708-3dbd9200-007c-11ea-9336-28bfc48be995.png">

**Comment by helenemccarron** [comment]
@brittany-johnson shouldn't the number (ie 42.) be on the same line? I remember the difficulty being the number not aligning to the top of the row (sorry for the highlights, they were distracting)

![2019-11-06_11-31-23](https://user-images.githubusercontent.com/8514591/68322406-fbe71880-0088-11ea-8cf6-7f537cc11bdc.png)

**Comment by brittany-johnson** [comment]
Ahh, I thought the problem with this issue was that that math was not aligning, I was looking at Alan's screenshots and how the 7/3 was not aligning properly. 
I didn't realize that this issue was supposed to address the alignment of the problem numbering. @helenemccarron

**Comment by helenemccarron** [comment]
The good news is, it looks much better when it's with-math that it did in my dingy screenshot ^_^
I think if necessary, adding padding around the svg might solve the issue. It is important to have the number and the body of the problem aligned though, so you might have to tweak it as well.

**Comment by helenemccarron** [comment]
![2020-01-09_14-51-04.png](https://images.zenhubusercontent.com/5b06d3244b5806bc2bd47abf/b817ddaa-ecb4-4d24-981f-edfca3c8d689)

This affects the following books:
anatomy
biology
ap-biology
chemistry

**Comment by helenemccarron** [comment]
The bold math is addressed in #1598

**Comment by marcjeannott** [comment]
Verified on the staging PDF of Chemistry

![2020-01-22_10-38-22.png](https://images.zenhubusercontent.com/5b572d7e0e34d640605a661c/653fa435-334d-4954-ae30-f47801cfd8a0)
